packages/data-transport-layer/src/services/l1-ingestion/handlers/sequencer-batch-appended.ts

```diff
@@ -152,6 +153,7 @@ export const handleEventsSequencerBatchAppended: EventHandlerSet<
           data: '0x',
           queueOrigin: 'l1',
           type: 'EIP155',
+          value: '0x0',
```

Comment:
I think `maybeDecodeSequencerBatchTransaction` also needs to be modified to parse the `value` field.

Reply:
The L1 to L2 txs have value 0, above `decoded.value` is used for queue origin sequencer transactions